src/app/components/AccountMenu/index.tsx

```diff
@@ -55,153 +58,213 @@ function AccountMenu({ showOptions = true }: Props) {
+                  )}
+                </span>
               </div>
             </div>
-            <PopiconsChevronBottomLine className="h-4 w-4" />
-            <span className="sr-only">{t("screen_reader")}</span>
+            <div className="flex flex-row gap-1 items-center">
+              {(isAlbyLNDHubAccount(
+                authAccount.alias,
+                authAccount.connectorType
+              ) ||
+                isAlbyOAuthAccount(authAccount.connectorType)) && (
+                <a
+                  className="cursor-pointer text-gray-600 dark:text-neutral-400 hover:text-gray-400 dark:hover:text-neutral-600"
+                  onClick={() => {
+                    window.open(`https://getalby.com/user`, "_blank");
+                  }}
+                  title={t("options.account.go_to_web_wallet")}
+                >
+                  <PopiconsGlobeLine className="w-5 h-5 mr-2 shrink-0" />
+                </a>
+              )}
+              <a
+                className="cursor-pointer text-gray-600 dark:text-neutral-400 hover:text-gray-400 dark:hover:text-neutral-600"
+                title={tCommon("wallet_settings")}
+                onClick={() => {
+                  openOptions(`accounts/${authAccount.id}`);
+                }}
+              >
+                <PopiconsSettingsMinimalLine className="w-5 h-5 mr-2 shrink-0" />
+              </a>
+            </div>
           </div>
-        </Menu.Button>
-        <Menu.List position="right" fullWidth>
-          {authAccount && (
-            <Menu.Item>
-              <div className="p-2 overflow-hidden">
-                <div className="flex flex-row items-center justify-between gap-2 bg-amber-50 dark:bg-brand-yellow/50 border-brand-yellow border-l-4 p-2 rounded-lg">
-                  <div className="flex flex-row items-center gap-2 overflow-hidden">
+        </div>
+      )
+    );
+  }
+
+  type ListItemProps = {
+    isPrompt?: boolean;
+  };
+
+  function ListItem({ isPrompt }: ListItemProps) {
+    return (
+      <>
+        <p className="flex justify-center text-lg dark:text-white p-3 font-semibold">
+          {t("select_wallet")}
+        </p>
+        {authAccount && (
+          <Menu.Item>
+            <ActiveAccount />
+          </Menu.Item>
+        )}
+        {Object.keys(accounts).length > 1 && (
+          <>
+            {Object.keys(accounts).map((accountId) => {
+              // Do not render the current active account
+              if (accountId === authAccount?.id) {
+                return;
+              }
+
+              const account = accounts[accountId];
+              return (
+                <Menu.ItemButton
+                  key={accountId}
+                  onClick={() => {
+                    selectAccount(accountId);
+                    if (window.location.pathname !== "/prompt.html") {
+                      navigate("/");
+                    }
+                    if (isPrompt) {
```

Comment:
Instead of having an isPrompt variable I think it would be better to allow instances to pass an onClick function, that is called here. (more generic)